pilot/pkg/model/context.go

```diff
@@ -139,6 +139,32 @@ func (node *Proxy) GetRouterMode() RouterMode {
 	return StandardRouter
 }
 
+// UnnamedNetwork is the default network that proxies in the mesh
+// get when they don't request a specific network view.
+const UnnamedNetwork = ""
+
+// GetNetworkView returns the networks that the proxy requested.
+// When sending EDS/CDS-with-dns-endpoints, Pilot will only send
+// endpoints corresponding to the networks that the proxy wants to see.
```

Comment:
So this setting applies to both egress and regular pods ( for example adding the env via auto-inject ? )

Reply:
Yes you can. By default regular pods have UnnamedNetwork view ("" network). And all existing endpoints are by default in the UnnamedNetwork (""). So things don't break.